tests/support/mod.rs

```diff
@@ -29,7 +29,7 @@ pub fn sink_failing_healthcheck() -> (Receiver<Event>, MockSinkConfig) {
 }
 
 pub fn source() -> (Sender<Event>, MockSourceConfig) {
-    let (tx, rx) = futures::sync::mpsc::channel(10);
+    let (tx, rx) = futures::sync::mpsc::channel(0);
```

Comment:
I believe this is still technically `1` slot since each sender gets a slot :( This was something that `tokio_sync::mpsc` fixed.